tests/System.Numerics.Tensors.Tests/TensorSliceTests.cs

```diff
@@ -0,0 +1,310 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections;
+using System.Linq;
+using Xunit;
+
+namespace System.Numerics.Tensors.Tests
+{
+    public class TensorSliceTests : TensorTestsBase
+    {
+        [Theory]
+        [MemberData(nameof(GetSingleTensorConstructors))]
+        public void SliceTest(TensorConstructor constructor)
+        {
+            var arr = Enumerable.Range(0, 24).ToArray();
+            var tensor = constructor.CreateFromArray<int>(arr).Reshape(new[] { 2, 3, 4 });
+
+            // default              reverse stride
+            //{{{  0, 1, 2, 3},         {{{ 0, 6,12,18},
+            //  {  4, 5, 6, 7},           { 2, 8,14,20},
+            //  {  8, 9,10,11}},          { 4,10,16,22}},
+            //
+            // {{ 12,13,14*,15*},        {{ 1, 7,13*,19*},
+            //  { 16,17,18*,19*},         { 3, 9,15*,21*},
+            //  { 20,21,22*,23*}}}        { 5,11,17*,23*}}}
+
+            // get the *'d elements above
+            var slice = tensor.Slice(new Range(1, 1), new Range(0, 3), new Range(2, 2));
```

Comment:
shouldn't it be 
var slice = tensor.Slice(new Range(1, 1), new Range(0, 3), new Range(2, 4));